test/MVCFramework45.FunctionalTests/FunctionalTest/DependencyTelemetryMvcTests.cs

```diff
@@ -11,12 +11,80 @@ namespace SampleWebAppIntegration.FunctionalTest
     using Microsoft.ApplicationInsights.Channel;
     using Microsoft.ApplicationInsights.DataContracts;
     using Xunit;
+    using Microsoft.ApplicationInsights.Extensibility;
+    using Microsoft.ApplicationInsights.DependencyCollector;
+    using Microsoft.Extensions.DependencyInjection;
 
     public class DependencyTelemetryMvcTests : TelemetryTestsBase
     {
         private const string assemblyName = "MVCFramework45.FunctionalTests";
 
         [Fact]
+        public void OperationIdOfOutgoingRequestIsPropagated()
```

Comment:
this supposed to be tested by Dependency Collector Module itself. The only test needed in this repo is that everything was configured correctly